src/App.css

```diff
@@ -2,10 +2,11 @@
 
 body {
   height: 100%;
-  width: 100%;
+  width: 99%;
```

Comment:
Why?

Reply:
There was a constant, unnecessary horizontal scroll bar that was the result of extra space. It also threw off the page symmetry on page load. I figured the only thing that could be asking for this much of the page was the body, so I tried this and it worked